### PR TITLE
User Invite: Add link blue color

### LINKED ISSUE #96629: Users: "Learn more" link in invite form is not blue
Issue reported during walkthrough: p58i-iKo-p2

The "Learn more" link that appears below the Role field in the user invite form should be blue.

Image

### PR DESCRIPTION
**Pull request by renancarvalho**
Related to https://github.com/Automattic/wp-calypso/issues/96629
Fixes https://github.com/Automattic/wp-calypso/issues/96629

## Proposed Changes

* Add theme blue color on link

## Why are these changes being made?

* Keeps UX consistent

## Testing Instructions

* Use live link or pull this branch
* Navigate to http://calypso.localhost:3000/people/new/{your_site} — Or live link instead of localhost
* Check if “learn more” is blue, it will get a blue color if clicked because of the :visited class that will run after clicked
image

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
    - [ ] For UI changes, have we tested the change in various languages (for example, ES, PT, FR, or DE)? The length of text and words vary significantly between languages. 
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/my-sites/people/team-invite/style.scss [hunk 1]
```diff
@@ -42,6 +42,9 @@
 		margin-bottom: 1rem;
 	}
 }
+.team-invite-form-learn-more__link {
+	color: var(--color-accent);
```
Comment (Aurorum) [inline client/my-sites/people/team-invite/style.scss]: Should this be `—color-link`? If I recall correctly, it’s slightly different on a few schemes like Sakura.

```suggestion
	color: var(--color-link);
```

Comment (renancarvalho) [reply client/my-sites/people/team-invite/style.scss]: Thank you, @Aurorum, I guess you're absolutely right :)

**Review by escapemanuele — APPROVED** [review]
LGTM

**Comment by Robertght** [comment]
@renancarvalho I noticed this was added to the One Board, so I'm following up here as we won't need PRs added there. Thank you!

**Pull request merged**